src/calculation/CompositeReportBuilder.ts

```diff
@@ -173,9 +246,6 @@ export class CompositeReportBuilder<T extends PopulationGroupResult> extends Abs
       if (patientDenomCount !== 0) {
         this.compositeFraction.numerator += (patientNumerCount * 1.0) / patientDenomCount;
       }
-    } else if (this.compositeScoringType === 'weighted') {
```

Comment:
Bring back this error but with updated message to indicate that `addPatientResults` cannot be used for weighted scoring right now and `addAllResults` should be used instead.